code/game/turfs/simulated.dm

```diff
@@ -79,7 +79,7 @@
 	tracks.AddTracks(bloodDNA,comingdir,goingdir,bloodcolor)
 
 /turf/simulated/proc/update_dirt()
-	dirt = min(dirt++, 101)
+	dirt = min(dirt+1, 101)

```

Comment:
People need to learn the difference between pre- and post-decrement. And for these purposes, the fix is better.